homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
```

Comment:
Opinionated comment on this: in sensor.py you now use `_handle_coordinator_update`, while this whole logic can be replaced with a 
```py
@property
def native_value(self) -> float:
    return getattr(self._lock, self.entity_description.key)
```
(Ideally we extend the sensor description and add a `value_fn` that gets the value as this is more type safe)

This removes the added logic of the `if self.device_id in self.coordinator.data.lock:` and simplifies a bit.